### PR TITLE
Create Setting for Next Episode Button Time

### PR DESCRIPTION
**Pull request by candry7731**
Creates a setting for users to customize when the next up button appears. 

The default is 30 seconds

To disable the button set to 0

### CODE REVIEW

**Comment by candry7731** [comment]
Adds #942 functionality

File: settings/settings.json [hunk 1]
```diff
@@ -86,7 +93,7 @@
                         "description": "Set the maximum amount of days a show should stay in the 'Next Up' list without watching it.",
                         "settingName": "ui.details.maxdaysnextup",
                         "type": "integer",
-                        "default": "365"
```
Comment (1hitsong) [inline settings/settings.json]: Why are we modifying the default value for maxdaysnextup?

File: settings/settings.json [hunk 2]
```diff
@@ -63,6 +63,13 @@
                 "settingName": "playback.subs.onlytext",
                 "type": "bool",
                 "default": "false"
+            },
+            {
+                "title": "Change next up button show duration",
```
Comment (1hitsong) [inline settings/settings.json]:
```suggestion
                "title": "Next Episode Button Time",
```

File: settings/settings.json [hunk 3]
```diff
@@ -63,6 +63,13 @@
...
+                "description": "Default is 30 seconds, you can set to 0 to diable the button or set new time in seconds to display the button",
```
Comment (1hitsong) [inline settings/settings.json]:
```suggestion
                "description": "Set how many seconds before the end of an episode the Next Episode button should appear. Set to 0 to disable.",
```

**Review by 1hitsong — CHANGES_REQUESTED** [review]
Also, new setting title and descriptions need to be added to translations file.

File: settings/settings.json (hunk 2 above)
Comment (sevenrats) [inline settings/settings.json]: Please insert new settings in alphabetical order. General Menu, Menus, Settings

Comment (candry7731) [reply settings/settings.json]: Done

File: settings/settings.json (hunk 1 above)
Comment (candry7731) [reply settings/settings.json]: That was a mistake, I fixed it back

File: settings/settings.json (hunk 2 above)
Comment (candry7731) [reply settings/settings.json]: This is updated

File: settings/settings.json (hunk 3 above)
Comment (candry7731) [reply settings/settings.json]: This is updated

**Review by 1hitsong — APPROVED** [review]

**Pull request merged**